tests/correctness/float.futil.data

```diff
@@ -0,0 +1,23 @@
+{
+    "inp": {
+        "data": [
+            15.1239
+        ],
+        "format": {
+            "numeric_type": "floating_point",
+            "is_signed": false,
+            "width": 32
+        }
+    },
+    "out": {
+        "data": [
+            0.0,
+            0.0
```

Comment:
should this be `0.56`?

Reply:
I got it, this is the input, not the expect file. My bad.

Reply:
Same as above